src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -725,10 +725,10 @@ public function prependExtensionConfig($name, array $config)
      *  * The parameter bag is frozen;
      *  * Extension loading is disabled.
      *
-     * @param bool $resolveEnvPlaceholders Whether %env()% parameters should be resolved using the current
+     * @param bool $resolveEnvPlaceholders whether %env()% parameters should be resolved using the current
```

Comment:
Not related to that PR, asked by fabbot https://fabbot.io/report/symfony/symfony/29409/d7908f9d83aebfc5a4e18e4fbc685827a78cccbd

Reply:
let's revert it - there's an issue to fix that as we consider it a false positive